exp-timing/zone2_scan.py

```diff
@@ -0,0 +1,54 @@
+import time
+import logging
+import sys
+from epics import caget, caput, cainfo, PV
+
+# setup PVs
+tgt_pv = 'LAS:LHN:LLG2:02:PHASCTL:DELAY_SET'
+tgt_time_pv = PV('LAS:LHN:LLG2:02:PHASCTL:DELAY_SET')
+ctr_time_pv = PV('LAS:LHN:LLG2:02:PHASCTL:GET_TIC_NS')
+
+#target/counter time dictionaries and initial values
+tgt_time = dict()
+ctr_time = dict()
+tgt = tgt_time_pv.value
+ctr = ctr_time_pv.value
+
+# scanning parameters 
+init_step = 0.05 # initial step size
+stop = 8 # total number of steps to take
+mult = 10 # multiplier by which to increase/decrease step size after each step
+wait_time = 5 # wait time in seconds b/t steps
+
+# time scan
+for x in range(0, stop):
+    if (x<(stop/2)):
+        caput(tgt_pv,tgt+init_step*(mult**x), wait=True) # step up with increasing step size
```

Comment:
This setup is kind of weird. Why are you using `caput` after already creating a `PV` class for the PV? You can just use `tgt_pg.put()`. At the beginning of the script you can instantiate your PV classes and then just use those throughout. 

https://pyepics.github.io/pyepics/pv.html#the-pv-class